torch/jit/annotations.py

```diff
@@ -342,7 +342,7 @@ def ann_to_type(ann, loc):
     the_type = try_ann_to_type(ann, loc)
     if the_type is not None:
         return the_type
-    raise ValueError(f"Unknown type annotation: '{ann}'")
+    raise ValueError(f"Unknown type annotation: '{ann}' at{loc.highlight()}")
```

Comment:
```suggestion
    raise ValueError(f"Unknown type annotation: '{ann}' at {loc.highlight()}")
```

Reply:
if you look at the output I got, there are already some spaces in `highlight()`. Is this change still needed?

Reply:
I'm not sure why there are already some spaces but it'll be good to have at least one in case there aren't spaces in the highlight.